features/fixtures/ios/build.sh

```diff
@@ -5,8 +5,44 @@ echo "$(sw_vers -productName) $(sw_vers -productVersion) $(sw_vers -buildVersion
 
 set -euxo pipefail
 
+disable_swizzling_key='<key>test_key_to_replace1</key>'
+disable_swizzling_value='<string>test_value_to_replace1</string>'
+swizzling_premain_key='<key>test_key_to_replace2</key>'
+swizzling_premain_value='<string>test_value_to_replace2</string>'
+
+fixture_name='Fixture'
+for ((i=1;i<=$#;i++));
+do
+    if [ ${!i} = '--fixtureName' ]
```

Comment:
Do you really need this option?  Could you just set the fixture name based on whether the other options are present?  E.g. if `--disableSwizzling` is passed then always call it `FixtureWithDisableSwizzling.ipa`

Reply:
I think it will be better to keep this parameter. As we add more parameters and allow different combinations for our tests, I think it might get tricky in the future to figure out the correct name inside of the script. Also, it is cleaner to set the `.ipa` name in the pipeline configuration, as it depends on the name being what it expects